aspnetcore/blazor/debug.md

```diff
@@ -93,11 +93,53 @@ While debugging a Blazor WebAssembly app, you can also debug server code:
 > [!NOTE]
 > Breakpoints are **not** hit during app startup before the debug proxy is running. This includes breakpoints in `Program.Main` (`Program.cs`) and breakpoints in the [`OnInitialized{Async}` methods](xref:blazor/components/lifecycle#component-initialization-methods) of components that are loaded by the first page requested from the app.
 
-# [Visual Studio Code](#tab/visual-studio-code)
+If the app is hosted at a different [app base path](xref:blazor/host-and-deploy/index#app-base-path) than `/`, update the following properties in `Properties/launchSettings.json` to reflect the app's base path:
+
+* `applicationUrl`:
+
+  ```json
+  "iisSettings": {
+    ...
+    "iisExpress": {
+      "applicationUrl": "http://localhost:{INSECURE PORT}/{APP BASE PATH}/",
+      "sslPort": {SECURE PORT}
+    }
+  },
+  ```
+
+* `inspectUri` of each profile:
+
+  ```json
+  "profiles": {
+    ...
+    "{PROFILE 1, 2, ... N}": {
+      ...
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/{APP BASE PATH}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      ...
+    }
+  }
+  ```
+
+The placeholders in the preceding settings:
+
+* `{INSECURE PORT}`: The insecure port. A random value is provided by default, but a custom port is permitted.
+* `{APP BASE PATH}`: The app's base path.
+* `{SECURE PORT}`: The secure port. A random value is provided by default, but a custom port is permitted.
+* `{PROFILE 1, 2, ... N}`: Launch settings profiles. Usually, an app specifies more than one profile by default (for example, a profile for IIS Express and a project profile, which is used by Kestrel server).
 
-<a id="vscode"></a>
+In the following examples, the app is hosted at `/OAT` with an app base path configured in `wwwroot/index.html` as `<base href="/OAT/">`:
```

Comment:
We should add a note here about how to configure the base path for Kestrel-based servers by modifying the Program.cs file.

Reply:
It's covered in the cross-linked content but a note is a good idea. I'll do it on the next commit.